testsuite/tests/kuadrant/defaults/test_rules_exclusivity.py

```diff
@@ -0,0 +1,37 @@
+"""Test mutual exclusivity of defaults block and implicit defaults"""
+
+import pytest
+from openshift_client import OpenShiftPythonException
+
+from testsuite.policy.rate_limit_policy import Limit
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, oidc_provider):
+    """Create AuthPolicy with basic oidc rules inside and outside defaults block"""
+    authorization.identity.add_oidc("inside-defaults", oidc_provider.well_known["issuer"], defaults=True)
+    authorization.identity.add_oidc("outside-defaults", oidc_provider.well_known["issuer"])
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def rate_limit(rate_limit):
+    """Add basic rate limiting rules inside and outside defaults block"""
+    rate_limit.add_limit("inside-defaults", [Limit(2, 5)], defaults=True)
+    rate_limit.add_limit("outside-defaults", [Limit(2, 5)])
+    return rate_limit
+
+
+@pytest.fixture(scope="module")
+def commit():
+    """We need to try to commit objects during the actual test"""
+    return None
+
+
+def test_rules_exclusivity(authorization, rate_limit):
```

Comment:
how about parametrizing it instead? In that wa, it will be two separate testcase and it will be much clearer what failed